app/domain/logging.go

```diff
@@ -0,0 +1,34 @@
+package logging
+
+import (
+	"encoding/json"
+	"io"
+	"log"
+)
+
+type Message struct {
```

Comment:
Entities are primarily responsible for passing data to the repository layer, so it seems that JSON annotations are not necessary.
It is better not to use JSON-tagged structs in the Domain layer since structures like Request Models and Response Models, which include JSON tags, should be used in the layer that handles client interactions.